src/schema/notifications.ts

```diff
@@ -41,6 +41,31 @@ function findDataId(
 	)[0];
 }
 
+function addNewItemToList(itemList: any, item: any, sortBy: any) {
+	if (sortBy === 'nameAsc' && itemList.length > 0) {
+		const index = itemList.findIndex(
+			(cnt: any) =>
+				item.fileAsStr.localeCompare(cnt.fileAsStr, undefined, {
+					sensitivity: 'base'
+				}) === -1
+		);
+		if (index !== -1) {
+			itemList.splice(index, 0, item);
+		} else {
+			itemList.push(item);
+		}
+	} else {
+		itemList = [item].concat(itemList);
```

Comment:
push

Reply:
Here, Initial approach was like this so kept same,

when the list is empty the 'searchResponse[query].contacts' is not coming as array so it's not getting pushed as expected to handle that it has written like that.

Reply:
it gives problem when the there is no list and we are adding a new item.